src/main/java/com/company/patterns/CustomGameFactory.java

```diff
@@ -0,0 +1,17 @@
+package com.company.patterns;
+
+public class CustomGameFactory implements GameAbstractFactory {
```

Comment:
any reason why you have 2 separate game factories and not just combine to into a single Factory file?